dev-vcs/git-pw/git-pw-2.0.0.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8..9} )
+DISTUTILS_USE_SETUPTOOLS=bdepend
```

Comment:
This is the [default](https://dev.gentoo.org/~mgorny/python-guide/distutils.html#distutils-use-setuptools), so we don't need to set it (it's not a problem if you do though)